capsules/extra/src/log.rs

```diff
@@ -872,7 +872,7 @@ impl<'a, F: Flash + 'static> flash::Client<F> for Log<'a, F> {
                     }
                 }
             }
-            flash::Error::FlashError | flash::Error::FlashMemoryProtectionError => {
+            Err(_) => {
```

Comment:
Here as well.

Reply:
Matched errors explicitly.

Reply:
Sometimes github hides things, did this one get updated?

Reply:
No, it didn't. I apologize. Now, it has been updated.